base/socket.jl

```diff
@@ -245,6 +245,7 @@ type InetAddr
     end
```

Comment:
this constructor is unnecessary now also. you might as well just delete both constructors since the default constructor has the same behavior already